apps/cli/src/fs-utils.ts

```diff
@@ -47,3 +39,46 @@ export const loadJSONFile = async <T>(filePath: string): Promise<T | null> => {
     return null;
   }
 };
+
+export const scaffoldProjectTemplate = async (
```

Comment:
Used only in build. Wouldn't call it "utils". Kinda core logic of build command. Let's move into build.ts